.circleci/config.yml

```diff
@@ -73,6 +73,15 @@ commands:
           command: ./gradlew androidDependencies
       - android/save-gradle-cache
 
+  install-ruby:
+    steps:
+      - run:
+          name: Install Ruby
+          command: |
+            apt-get update
+            (yes || true) | apt-get install ruby-full
+            gem install bundler
```

Comment:
So I thought about installing rvm instead of ruby directly... But I didn't want to spend too much time on this and I think this is acceptable. Unfortunately, it's needed to execute fastlane commands in the gcli executor.

Reply:
I'm confused - we already have ruby in our machines, that's what we use to run fastlane in the first place, right? 
Is there an issue with ruby versions?

Reply:
ruby is installed in the android docker machine that CircleCI provides, but not in the google cloud executor we use to execute the integration tests in firebase test labs. We could change the executor, but then we would need to install the gcloud cli to be able to run the tests there instead and add some extra code to handle that...

Reply:
ohh, I see, nvmd then let's keep this